src/libraries/System.Data.Common/src/System/Data/Common/DBCommandBuilder.cs

```diff
@@ -477,8 +477,7 @@ public virtual string QuoteSuffix
         {
             get
             {
-                string? quoteSuffix = _quoteSuffix;
-                return ((null != quoteSuffix) ? quoteSuffix : string.Empty);
+                return (_quoteSuffix ?? string.Empty);
```

Comment:
any particular reason for parens?

Reply:
Nope, they just aren't removed by the auto-fixer for the rule.  I removed them in cases like this as I was reviewing the changes if I noticed them.  There's a separate analyzer for removing unnecessary parens, which we're not ready for yet :)